src/Api/Controller/ShowDiscussionController.php

```diff
@@ -62,8 +68,9 @@ class ShowDiscussionController extends AbstractShowController
      * @param \Flarum\Discussion\DiscussionRepository $discussions
      * @param \Flarum\Post\PostRepository $posts
      */
-    public function __construct(DiscussionRepository $discussions, PostRepository $posts)
+    public function __construct(SlugManager $slugManager, DiscussionRepository $discussions, PostRepository $posts)
```

Comment:
New attributes should always go to the end OR be sorted alphabetically:

- to the end because overrides need less changes, more extensibility
- alphabetically  because that seems to be the convention?!

Reply:
Yeah agreed, good catch